importApp/src/main/java/com/commercetools/dataimport/orders/OrderCsvLineValue.java

```diff
@@ -0,0 +1,101 @@
+package com.commercetools.dataimport.orders;
+
+
+/**
+ * This class is used as a container for the csv line data
+ */
+
+public class OrderCsvLineValue {
+
+    private String customerEmail;
+    private String orderNumber;
+    private LineItems lineItems;
+    private double totalPrice;
+
+
+    public OrderCsvLineValue() {
+    }
+
+    public String getCustomerEmail() {
+        return customerEmail;
+    }
+
+    public void setCustomerEmail(String customerEmail) {
+        this.customerEmail = customerEmail;
+    }
+
+    public String getOrderNumber() {
+        return orderNumber;
+    }
+
+    public void setOrderNumber(String orderNumber) {
+        this.orderNumber = orderNumber;
+    }
+
+    public LineItems getLineItems() {
+        return lineItems;
+    }
+
+    public void setLineItems(LineItems lineItems) {
+        this.lineItems = lineItems;
+    }
+
+    public double getTotalPrice() {
+        return totalPrice;
+    }
+
+    public void setTotalPrice(double totalPrice) {
+        this.totalPrice = totalPrice;
+    }
+
+    public static class LineItems {
+
+        private Variant variant;
+        private double price;
+        private long quantity;
+
+        public double getPrice() {
+            return price;
+        }
+
+        public void setPrice(double price) {
+            this.price = price;
+        }
+
+        public long getQuantity() {
+            return quantity;
+        }
+
+        public void setQuantity(long quantity) {
+            this.quantity = quantity;
+        }
+
+
+        public Variant getVariant() {
+            return variant;
+        }
+
+        public void setVariant(Variant variant) {
+            this.variant = variant;
+        }
+
+        public static class Variant {
```

Comment:
I guess you wanted to answer the other comment ;)

If it wouldn't need the casting it would better, but for my taste casting is less readable than lambda (which is actually readable, just a bit verbose for a simple case like this) and a bit dangerous, specially since there might raise a warning and we don't see it via Github interface (it's an example, I know there isn't in this case).

Lambdas themselves are perfectly fine, method reference is just a particular case of lambdas. If the lambda is not "method reference" convertible is not an issue, it's simply not applicable everywhere.

But I'm fine with anything you choose there, as I said it's a subjective matter ;)